tcms/profiles/tests.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=invalid-nameg
```

Comment:
You have a typo here. Always check your pylint log after fixing issues, because you may have introduced new warnings by fixing the old ones. In this case it would tell you that this is an invalid warning name.